Nit changes in wording (#48)
huggingface/hf-endpoints-documentation: docs/source/guides/custom_handler.mdx

### 6. Deploy your Custom Handler as an Inference Endpoint

> The last step is to deploy your Custom Handler as an Inference Endpoint. You can deploy your Custom Handler like a regular Inference Endpoint. Add your repository, select your cloud and region, your instance and security setting, and deploy.

Reviewer: ```suggestion
The last step is to deploy your Custom Handler as an Inference Endpoint. You can deploy your Custom Handler like you would a regular Inference Endpoint. Add your repository, select your cloud and region, your instance and security setting, and deploy.
```

> The Inference Endpoint Service will check when creating your Endpoint if there is a `handler.py` available and valid and will use it for serving requests no matter which “Task” you select.

Reviewer: ```suggestion
When creating your Endpoint, the Inference Endpoint Service will check for an available and valid `handler.py`, and will use it for serving requests no matter which “Task” you select.
```
